ui/src/app/applications/components/application-node-info/application-node-info.tsx

```diff
@@ -84,6 +87,16 @@ export const ApplicationNodeInfo = (props: {
                 attributes.push({title: 'HEALTH DETAILS', value: props.controlled.summary.health.message});
             }
         }
+    } else if (props.node && (props.node as ResourceTreeNode).health) {
+        const treeNode = props.node as ResourceTreeNode;
+        attributes.push({
```

Comment:
Can we wrap this in:

```typescript
if (treeNode && treeNode.health) {
  ...
}
```
to guard against crashes?

Reply:
I updated it with the wrapping, although it feels a bit redundant as I'm checking the same logic as line 90